src/executables/master_server_monitor.hpp

```diff
@@ -0,0 +1,114 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * master_server_monitor.hpp -
+ */
+
```

Comment:
```suggestion
#ifndef _MASTER_SERVER_MONITOR_HPP_
#define _MASTER_SERVER_MONITOR_HPP_
```
파일 마지막 라인에 #endif 추가해주세요.
중복 및 순환 참조 막아주세요

Reply:
`#endif /* _MASTER_SERVER_MONITOR_HPP_ */` 형식으로 추가

Reply:
추가하였습니다.